tests/sentry/api/test_event_search.py

```diff
@@ -410,28 +410,42 @@ class RemapVisitor(SearchVisitor):
 
     def test_numeric_filter(self):
         # test numeric format
-        assert parse_search_query('some_number:>500') == [
+        assert parse_search_query('times_seen:500') == [
             SearchFilter(
-                key=SearchKey(name='some_number'),
+                key=SearchKey(name='times_seen'),
+                operator="=",
+                value=SearchValue(raw_value=500),
+            ),
+        ]
+        assert parse_search_query('times_seen:>500') == [
+            SearchFilter(
+                key=SearchKey(name='times_seen'),
                 operator=">",
                 value=SearchValue(raw_value=500),
             ),
         ]
-        assert parse_search_query('some_number:<500') == [
+        assert parse_search_query('times_seen:<500') == [
             SearchFilter(
-                key=SearchKey(name='some_number'),
+                key=SearchKey(name='times_seen'),
                 operator="<",
                 value=SearchValue(raw_value=500),
             ),
         ]
         # Non numeric shouldn't match
-        assert parse_search_query('some_number:<hello') == [
+        assert parse_search_query('times_seen:<hello') == [
             SearchFilter(
-                key=SearchKey(name='some_number'),
+                key=SearchKey(name='times_seen'),
                 operator="=",
                 value=SearchValue(raw_value="<hello"),
             ),
         ]
+        assert parse_search_query('times_seen:<512.1.0') == [
+            SearchFilter(
+                key=SearchKey(name='times_seen'),
+                operator="=",
+                value=SearchValue(raw_value="<512.1.0"),
```

Comment:
🤔 is this consistent with existing issue search behavior? seems like it would maybe be better to raise an error or something so the user knows what's wrong / can correct their query

Reply:
At the moment it raises an unhandled exception. I'm not sure what the ideal behaviour is... I can see for an integer that it's best to raise an invalid format exception, but for a string/tag maybe it's fine?

Points again to us needing our fields to be typed. I'll tackle that once I get this search out in general though.